ECommerceAPI/Services/AuthenticationServices/Implementations/AuthenticationService.cs

```diff
@@ -102,12 +103,67 @@ public async Task<ErrorOr<LoginResponseDto>> RefreshTokenAsync(string refreshTok
             var result =await tokenService.GenerateTokenAsync(user);
             return result;
         }
+        public async Task<ErrorOr<SuccessResponse>> ForgotPasswordAsync(string email)
+        {
+            var user = await identityService.FindByEmailAsync(email);
+            if (user is null)
+            {
+                return IdentityErrors.UserNotFound;
+            }
+            var baseUrl = "https://localhost:5001/reset-password";// just for testing
```

Comment:
move any static URL to shared place please

Reply:
done